### DISCUSSION TITLE
Version info data

### DISCUSSION THREAD

**Discussion by 4O4A** [wavelog/wavelog#725] [Q&A] [answered]
I upgraded manually from 1.5.2 to 1.8.2. and cleared cache in browser.

When I check version info, it still reads 1.5.2.
![image](https://github.com/user-attachments/assets/8a94b8b3-95b3-42f0-b18a-97c77dbaf22e)

Where is this data pulled from?

**Comment by HB9HIL** [comment]
Can you please visit the "Debug Information" view and check which "Wavelog Version" and wich "Migration" is shown there?

Please also make sure you take advantage of this: https://github.com/wavelog/wavelog/discussions/665

**Comment by 4O4A** [comment]
Debug information page is giving me error 500.

I'm not sure about #656 since I simply overwrite files from old version to new version.

Edit: In that user menu, Debug and Cron Manager are returning 500 error, other menu items are fine
More edit: In Awards, DOK, JCC, WAJA... return error 500. WAB, WAS is OK.

**Comment by HB9HIL** [comment]
If you are not using `git` you can ignore #665

Please check the following:
- Turn on Error logging in `application/config/config.php` (Level 1 should be enough: `$config['log_threshold'] = 1;`) and reload the debug view
- Check the migration version in `application/config/migration.php` and tell me (do not edit this file!)

**Reply by HB9HIL** [reply]
PS: You can find the log files in `application/logs`

**Reply by 4O4A** [reply]
Thanks, here are results:

- Error log has only one line: "Directory access is forbidden."
- Migration version reads: `$config['migration_version'] = 213;`

**Reply by HB9HIL** [reply]
You can't access the error log from the browser. If you have a FTP hosting you have to download the file.

**Reply by 4O4A** [reply]
That's exactly what I did, that's the content of file.

HTML code:

![image](https://github.com/user-attachments/assets/539ed854-9985-41fd-ae94-bd2e270efb1d)

**Reply by 4O4A** [reply]
Orrrrr... this mean that nothing is generated in /logs folder?

**Reply by 4O4A** [reply]
Looks like nothing in /logs folder. 

![image](https://github.com/user-attachments/assets/5964adbd-660d-458c-a25d-39fb3230b6ac)

**Reply by 4O4A** [reply]
Nothing happens even with log threshold set to 4, no log is generated. Now I see in code how file name should look like, but folder is still empty (besides index.html file).

`$config['log_threshold'] = 4;`

**Reply by HB9HIL** [reply]
So you changed the value of `log_threshold` in config.php and tried to reload the page and nothing happened?

**Reply by 4O4A** [reply]
Yes, nothing happened.

**Comment by phl0** [comment]
Wrong permissions for logs folder?

**Reply by 4O4A** [reply]
Changed it to 777 -- no joy.

**Reply by phl0** [reply]
What about parent folder application?

**Reply by 4O4A** [reply]
No, even though both are 777.

**Comment by 4O4A** [comment]
I have an error log in root of installation.
These messages repeat during the time:

[10-Aug-2024 11:20:32 America/Chicago] PHP Warning:  Undefined array key "log_path" in /home/xxxxx/public_html/sat/system/core/Log.php on line 127
[10-Aug-2024 11:20:32 America/Chicago] PHP Warning:  Undefined array key "log_path" in /home/xxxxx/public_html/sat/system/core/Log.php on line 128
[10-Aug-2024 11:20:32 America/Chicago] PHP Warning:  Undefined array key "log_threshold" in /home/xxxxx/public_html/sat/system/core/Log.php on line 140
[10-Aug-2024 11:20:32 America/Chicago] PHP Warning:  Undefined array key "log_threshold" in /home/xxxxx/public_html/sat/system/core/Log.php on line 144

**Comment by HB9HIL** [comment, marked as answer]
For future reference:

4O4A and I tracked down this issue and we came to the result that his MySQL version 5.6.51 was too old for one of the migrations. We gonna fix this to support older MySQL versions. 

if you come here with the same Issue please upgrade to a version HIGHER then 1.8.2 as we fixed the migration to be a bit more backwarts compatible. (fix [here](https://github.com/wavelog/wavelog/commit/75b86461ea905c8f77df7775f3ed4456db3d207c))
